ckanext/canada/controller.py

```diff
@@ -158,45 +166,42 @@ def logged_in(self):
         else:
             h.flash_error(_('Login failed. Bad username or password.'))
             return h.redirect_to(controller='user',
-                action='login', locale=lang)
+                                 action='login', locale=lang)
```

Comment:
Brackets should always open and close at the same level of indentation.

Aggressively indent the first line instead of every subsequent line.

```
return h.redirect_to(
    controller='user',
    ....
)
```